libs/openchallenges/ui/src/lib/paginator/paginator.component.ts

```diff
@@ -9,22 +18,28 @@ import { PaginatorModule as PrimengPaginatorModule } from 'primeng/paginator';
   templateUrl: './paginator.component.html',
   styleUrls: ['./paginator.component.scss'],
 })
-export class PaginatorComponent implements OnInit {
+export class PaginatorComponent {
   @Input({ required: true }) pageNumber = 0; // index of the new page
   @Input({ required: false }) pageLinkSize = 5;
   @Input({ required: true }) pageSize = 0; // number of rows to display in new page
```

Comment:
Is the comment correct? The page size is usually a number of items in a page, not a number of rows.

Reply:
>Is the comment correct? The page size is usually a number of items in a page, not a number of rows.

In `primeng` terms, they use "rows" to represent "page". I will update the comment to match our OC language : )